docs/.eleventy.js

```diff
@@ -49,6 +52,7 @@ module.exports = function(eleventyConfig) {
 
     eleventyConfig.addGlobalData("site_name", siteName);
     eleventyConfig.addGlobalData("GIT_BRANCH", process.env.BRANCH);
+    eleventyConfig.addGlobalData("HEAD", pathPrefix === "/docs/head/");
```

Comment:
It's probably safer to check the git branch

```suggestion
    eleventyConfig.addGlobalData("HEAD", GIT_BRANCH === "main");
```

Reply:
For deploy previews or in local development it may not be `main`, so updated it to `branch !== 'latest' || branch !== 'next'`